doc/nrf/config_and_build/dfu/nrf70_fw_patch_update.rst

```diff
@@ -0,0 +1,207 @@
+=============
+
+To use this feature, ensure that the following prerequisites are met:
+
+* The external memory is available and properly configured in the device tree.
+* The external memory has sufficient capacity to accommodate the firmware patches.
+  This includes additional space for potential patch upgrades, such as those required for DFU.
+  The combined size of all firmware patches should not exceed 128 kB.
+* MCUboot is enabled, and the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option is set to ``y``.
+
+Supported platforms
+===================
+
+The following platforms are supported:
+
+* nRF5340 DK with nRF7002 EK as a shield
+
+Preparing the application
+=========================
+
+The DFU procedure for the nRF70 Series firmware patch, when located in external memory, requires the creation of additional MCUboot partitions.
+This can be accomplished using the Partition Manager by defining the necessary partition layout in the Partition Manager configuration file specific to your application.
+For instructions on configuring basic partitions with the Partition Manager, refer to the :ref:`partition_manager` documentation.
+
+The instructions in this guide assume you are using the Partition Manager and its associated configuration files.
+
+.. _nrf70_fw_patch_update_mcuboot_partitions:
+
+Defining nRF70 series MCUBoot partitions
+----------------------------------------
+
+The number of MCUboot partitions required depends on the platform being used and its memory layout.
+In addition to a partition dedicated to the nRF70 Series firmware patch, you will need to create specific partitions based on the platform:
+
+* For applications that are based on a single image, define a new partition with the number ``1``.
+* For multi-image builds, define the subsequent partition to follow the last existing one.
+
+Below are examples of MCUboot partition names for updating the nRF70 Series firmware patch, which vary depending on the platform and the number of cores used:
+
+* For the nRF5340 DK in a single-core variant (without the network core): ``mcuboot_primary_1`` and ``mcuboot_secondary_1``.
+* For the nRF5340 DK in a multi-core variant (with the network core): ``mcuboot_primary_2`` and ``mcuboot_secondary_2``.
+
+.. _nrf70_fw_patch_update_adding_partitions:
+
+Adding nRF70 series firmware patch partitions
+---------------------------------------------
+
+The examples below assume that there are two existing MCUboot partitions (for the application and network cores) and that the starting address of the free external memory space is ``0x12f000``.
+
+To add the required partitions for the nRF70 Series firmware patch update, complete the following steps:
+
+1. Create the ``nrf70_wifi_fw_mcuboot_pad`` partition for the MCUboot header.
+
+   This partition should start from the first available address in the external memory space and have a size equal to the value of the :kconfig:option:`CONFIG_IMG_BLOCK_BUF_SIZE` Kconfig option.
```

Comment:
this is not correct

Reply:
What is not correct in this sentence?

Reply:
What relation does the pad section size have to do with the img block buffer size?

Reply:
Ok, the (pad + image) size must be aligned, right?

Reply:
the start and end addresses must be aligned, the header size must be the size of the mcuboot header